client/src/ui/molecules/theme-toggle/index.scss

```
@@ -3,3 +3,11 @@
   align-items: center;
   gap: 0.2rem;
 }
+
+.icon-moon-fill {
+  transform: rotate(35deg);
+}
+
+.icon-sun-fill {
+  margin-inline-end: 0.2rem;
+}
```

It wouldn't hurt to have the same margin for the moon icon, and to make it specific to the toggle's icon, rather than adding it to the icon of that type in general:

```suggestion
.theme-toggle .icon {
  margin-inline-end: 0.2rem;
}
```